### DISCUSSION TITLE
Language name is different in Chrome on PC

### DISCUSSION THREAD

**Discussion by Tykmis** [adityatelange/hugo-PaperMod#617] [Q&A]
Hi, 

I noticed a problem with the language flag in chrome and edge (i think its a problem in chromium based web browsers).

Edge/Chrome: 
![image](https://user-images.githubusercontent.com/87838891/140105574-9e26e28c-b5bd-4dd1-a347-005be532c8c9.png)

Firefox:
![image](https://user-images.githubusercontent.com/87838891/140105665-053081a3-8f03-4d0b-9e41-74a18e3b6520.png)

On moblie Chrome (android), everything is fine (like a firefox on PC)
The same problem is also on my site. What you think? Its a problem with Papermod, Chromium engine or hugo? 
Is there a solution to this? The flag is prettier, adds color and is more eye-catching

Thanks for answers :)

**Comment by adityatelange** [comment]
I am not sure why this happened but I do not have this issue on browsers that I have tested with on Ubuntu Os and Android 10 both on Firefox as well as Chrome

**Comment by devmedeiros** [comment]
I've looked up this issue and learned that Google (and other chromium browsers) does this by default and it suggest that the web page have its own "emoji", I've been able to solve this by using this module: https://github.com/jakejarvis/hugo-mod-twemoji

This module converts all emojis in the web site to [Twemoji](https://twemoji.twitter.com/) (Twitter Emoji).
